util/src/main/java/org/datacommons/util/SummaryReportGenerator.java

```diff
@@ -70,10 +85,51 @@ public int getNumObservations() {
     public List<Double> getSeriesValues() {
       return this.seriesValues;
     }
-  }
 
-  public static boolean TEST_mode = false;
-  public static final String SUMMARY_REPORT_HTML = "summary_report.html";
+    public String getTimeSeriesChartSVG() {
+      TimeSeriesCollection dataset = new TimeSeriesCollection();
+      TimeSeries timeSeries = new TimeSeries("ts");
+      for (int i = 0; i < this.seriesDates.size(); i++) {
+        if (i >= this.seriesValues.size()) break;
+        LocalDateTime localDateTime = StringUtil.getValidISO8601Date(seriesDates.get(i));
+        if (localDateTime == null) continue;
+        Date date = Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant());
```

Comment:
What is ZoneId.systemDefault()?  If it varies between india vs. US, we can have weird diffs depending on where the tool is run, so should we pick UTC?

Reply:
in the next line, where we create a new [Day](https://www.jfree.org/jfreechart/api/javadoc/org/jfree/data/time/Day.html), the construction of that Day object is done based on the default time zone. Looking at the documentation for Day, I think it might actually be better to just pass in the day, month, year of the localDateTime rather than converting to Date first.